sources/packages/backend/libs/utilities/src/config/config.service.ts

```diff
@@ -154,6 +154,50 @@ export class ConfigService {
     );
   }
 
+  /**
+   * MAXIMUM IDLE TIME FOR WARNING STUDENT Configuration.
```

Comment:
Please review all the comments, I believe they are copied/pasted.
Besides that, no other comment in the file uses the all upper case env variable name. If possible let's try to make the comments similar to the other ones.